### PR TITLE
Support multi file inputs from S3 integration

### PR DESCRIPTION
**Pull request by cw75**
## Describe your changes and why you are making these changes
This PR enables user's S3 integration to extract multiple files in a concatenated form, so when they have multiple files they don't need to define a bunch of extract operators. The expected value for the SDK API is commented inline. The PR also adds an authentication step when the user registers an S3 integration by trying to list the files in the bucket.

https://www.loom.com/share/b692707630a44f3bbf0ebfa107e6334b

## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [x] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [x] I have run the integration tests locally and they are passing.
- [x] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [x] All features on the UI continue to work correctly.
- [ ] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: sdk/aqueduct/integrations/s3_integration.py [hunk 1]
```diff
@@ -66,7 +73,9 @@ def file(
                             extract=ExtractSpec(
                                 service=integration_info.service,
                                 integration_id=integration_info.id,
-                                parameters=S3ExtractParams(filepath=filepath, format=format),
+                                parameters=S3ExtractParams(
+                                    filepath=json.dumps(filepath), format=format
```
Comment (saurav-c) [inline sdk/aqueduct/integrations/s3_integration.py:77]: Can we document this somewhere that filepath needs to be JSON serialized

**Review by saurav-c — APPROVED** [review]
LGTM overall!

A couple of nits:
- we may want to rename `filepath` to `filepaths` to make it clear that we can multiple S3 paths
- did you figure out whether or not we need to specify an AWS region or does the default work in most cases?

File: sdk/aqueduct/integrations/s3_integration.py (hunk 1 above)
Comment (cw75) [reply sdk/aqueduct/integrations/s3_integration.py]: documented.

**Comment by cw75** [comment]
@saurav-c I changed the SDK external API to have `filepaths`, but didn't change the inner ones since that may require a data migration due to json tag name change. For the region, S3 bucket actually needs to be globally unique, and I verified that the region doesn't matter when initializing boto resource.

**Comment by vsreekanti** [comment]
Thanks for the video, @cw75 -- super helpful! Good catch on the incorrect validations. This mostly look good. Just one UI nit: 
* Rather than printing a JSON blob that's a little tough to read, could we please grab the error field out of the JSON blob and then use a preformatted text block (i.e., the `<pre> </pre>`) tag to format the error message? It will make it way easier to read. I believe we have an example of doing this in the log viewer component if you need a pointer.

Let's also make sure we clearly document the prefix search/nesting thing.

**Comment by cw75** [comment]
@vsreekanti Does this look fine now?
Screen Shot 2022-07-13 at 6 32 13 PM

**Pull request merged**